components/frontend/src/widgets/Button.js

```diff
@@ -99,28 +101,38 @@ export function AddDropdownButton({ itemSubtypes, itemType, onClick }) {
                 >
                     <Dropdown.Menu>
                         <Dropdown.Header>{`Available ${itemType} types`}</Dropdown.Header>
-                        {itemSubtypes.length > 5 && (
-                            <>
-                                <Dropdown.Divider />
-                                <Input
-                                    className="search"
-                                    focus
-                                    icon="search"
-                                    iconPosition="left"
-                                    onChange={(_event, { value }) => setQuery(value)}
-                                    onClick={(event) => {
-                                        event.stopPropagation()
-                                    }}
-                                    onKeyDown={(event) => {
-                                        if (event.key === " ") {
-                                            event.stopPropagation() // Prevent space from closing menu
-                                        }
-                                    }}
-                                    ref={inputRef}
-                                    placeholder={`Filter available ${itemType} types`}
-                                    value={query}
-                                />
-                            </>
+                        <Dropdown.Divider />
+                        <Input
+                            className="search"
+                            focus
+                            icon="search"
+                            iconPosition="left"
+                            onBlur={(event) => event.stopPropagation()}
```

Comment:
Does this mean that the filter option is blurred when selecting from all? What if you want to search through all the possible metrics?

Reply:
It's to prevent the input from being cleared when the user tabs to the checkbox.